satpy/readers/file_handlers.py

```diff
@@ -98,6 +99,22 @@ def combine_info(self, all_infos):
                                       'satellite_latitude',
                                       'satellite_altitude'))
 
+        # Average orbital parameters
+        orb_params = [info.get('orbital_parameters', {}) for info in all_infos]
+        if all(orb_params):
+            # Collect all available keys
+            orb_params_comb = {}
+            for d in orb_params:
+                orb_params_comb.update(d)
+
+            # Average known keys
+            keys = ['projection_longitude', 'projection_latitude', 'projection_altitude',
+                    'satellite_nominal_longitude', 'satellite_nominal_latitude',
+                    'satellite_actual_longitude', 'satellite_actual_latitude', 'satellite_actual_altitude',
+                    'nadir_longitude', 'nadir_latitude']
+            orb_params_comb.update(self._combine(orb_params, np.mean, *keys))
+            new_dict['orbital_parameters'] = orb_params_comb
+
```

Comment:
Handling of other parameters such as TLE data would have to be added in the future